src/examine_quality_filter.py/Repetitious_text.py

```diff
@@ -0,0 +1,115 @@
+"""
```

Comment:
Any reason for the new `src` folder? :D

Reply:
I actually went back and forth on this.

Basically, we kinda want two different code bases one which is simply the application, acting mostly as reference and documentation and the package which is intended to be reuseable.

Reply:
I would love if you have an opinion on this, in other forks I have actually done it using src/dfm and src/application